src/main/java/com/revature/services/JMeterServices.java

```diff
@@ -0,0 +1,247 @@
+package com.revature.services;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import com.revature.models.ResultSummary;
+import com.revature.responsecollector.JMeterResponseCollector;
+import com.revature.templates.LoadTestConfig;
+import io.swagger.models.HttpMethod;
+import io.swagger.models.Operation;
+import io.swagger.models.Path;
+import io.swagger.models.Swagger;
+import io.swagger.models.parameters.Parameter;
+import io.swagger.models.parameters.PathParameter;
+import org.apache.jmeter.control.LoopController;
+import org.apache.jmeter.engine.StandardJMeterEngine;
+import org.apache.jmeter.protocol.http.sampler.HTTPSampler;
+import org.apache.jmeter.reporters.Summariser;
+import org.apache.jmeter.testelement.TestElement;
+import org.apache.jmeter.testelement.TestPlan;
+import org.apache.jmeter.threads.SetupThreadGroup;
+import org.apache.jmeter.util.JMeterUtils;
+import org.apache.jorphan.collections.HashTree;
+import org.springframework.stereotype.Service;
+
+@Service
+public class JMeterServices {
```

Comment:
This is minor, but this class might be more aptly named as just singular 'JMeterService'.